src/tests/Equinor.ProCoSys.IPO.WebApi.IntegrationTests/appsettings.json

```diff
@@ -18,5 +18,23 @@
   "Synchronization": {
     "UserOid": "00000000-0000-0000-0000-000000000000",
     "Interval": "24:0:0"
+  },
```

Comment:
do we need this? Is it used in int tests?

Reply:
I tried without and the attachment tests fails ... hm .. I don't understand why IPO need this and preservation not. We can have a quick chat about it, but not use to much time

Reply:
I see why IPO need it. OK here. However, I don't understand why preservation to not need the Synchronization block in IntegrationTests/appsettings.json. I resolve this comment